packages/connectors/connector-gitlab/src/constant.ts

```diff
@@ -0,0 +1,36 @@
+import type { ConnectorMetadata } from '@logto/connector-kit';
+import { ConnectorPlatform } from '@logto/connector-kit';
+import { clientSecretFormItem, clientIdFormItem, scopeFormItem } from '@logto/connector-oauth';
+
+export const jwksUri = 'https://gitlab.com/oauth/discovery/keys';
+export const authorizationEndpoint = 'https://gitlab.com/oauth/authorize';
+export const userInfoEndpoint = 'https://gitlab.com/oauth/userinfo';
+export const tokenEndpoint = 'https://gitlab.com/oauth/token';
+export const scope = 'openid';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'gitlab-universal',
+  target: 'gitlab',
+  platform: ConnectorPlatform.Universal,
+  name: {
+    en: 'GitLab',
+  },
+  logo: './logo.svg',
+  logoDark: null,
+  description: {
+    en: 'GitLab is an online community for software development and version control.',
+  },
+  readme: './README.md',
+  formItems: [
+    clientIdFormItem,
+    clientSecretFormItem,
+    {
+      ...scopeFormItem,
+      placeholder: 'openid [profile email]',
```

Comment:
This placeholder seems unusual, we can align with other connectors.

Reply:
I was hesitant on it for that same reason, I’ll remove it and leave it as it is on other connectors

Reply:
Done, I've removed this line and left the default 
<img width="643" alt="image" src="https://github.com/user-attachments/assets/a31955b0-fcee-4bfb-a8ea-98423987e1c9">